### DISCUSSION TITLE
point_stat for Wind Profile

### DISCUSSION THREAD

**Discussion by wangrenz** [dtcenter/METplus#1934] [Statistical Computation] [answered]
Hi
I want to do `PROFLR` verify, but PROFLR obs data only have height, UGRD, VGRD. WRF output use UPP interpolated at pressure level. but point_stat is empty.

My obs data as follow, I use `ascii2nc` convert `little r` to ob.nc:
```
ncdump -v hdr_typ_table ob.nc

// global attributes:
                :MET_Obs_version = "1.02" ;
                :use_var_id = "false" ;
                :FileOrigins = "File / generated 20221113_052833 UTC on host node03 by the MET ascii2nc tool" ;
                :MET_version = "V10.1.2" ;
                :MET_tool = "ascii2nc" ;
data:

 hdr_typ_table =
  "PROFLR" ;
}

ncdump -v obs_hgt ob.nc

data:

obs_hgt = 
    100, 100, 100, 100, 100, 200, 200, 200, 200, 200, 300, 300, 300, 300,
    300, 400, 400, 400, 400, 400, 500, 500, 500, 500, 500, 600, 600, 600,
    600, 600, 700, 700, 700, 700, 700, 800, 800, 800, 800, 800, 900, 900,
    900, 900, 900, 1000, 1000, 1000, 1000, 1000, 1100, 1100, 1100, 1100,
    1100, 1200, 1200, 1200, 1200, 1200, 1300, 1300, 1300, 1300, 1300, 1400,
    1400, 1400, 1400, 1400, 1500, 1500, 1500, 1500, 1500, 1600, 1600, 1600,
    1600, 1600, 1700, 1700, 1700, 1700, 1700, 1800, 1800, 1800, 1800, 1800,
    1900, 1900, 1900, 1900, 1900, 2000, 2000, 2000, 2000, 2000, 2100, 2100,
    2100, 2100, 2100, 2200, 2200, 2200, 2200, 2200, 2300, 2300, 2300, 2300,
    2300, 2400, 2400, 2400, 2400, 2400, 2500, 2500, 2500, 2500, 2500, 2600,
    2600, 2600, 2600, 2600, 2700, 2700, 2700, 2700, 2700, 2800, 2800, 2800,
    2800, 2800, 2900, 2900, 2900, 2900, 2900, 3000, 3000, 3000, 3000, 3000 ;

ncdump -v obs_lvl ob.nc

data:

obs_lvl = 
    _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _,
    _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _,
    _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _,
    _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _,
    _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _,
    _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _,
    _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _,
    _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _, _ ;
```

My PointStatConfig as follow:
```
fcst = {
   file_type = GRIB2;
   field = [
      {
        message_type = [ "PROFLR" ];
        sid_exc      = [];
        name         = "UGRD";
        level        = [ "Z400", "Z500", "Z700", "Z850", "Z925", "Z1000" ];
      },
      {
        message_type = [ "PROFLR" ];
        sid_exc      = [];
        name         = "VGRD";
        level        = [ "P400", "P500", "P700", "P850", "P925", "P1000" ];
      }
...
obs = fcst;
```

level set `Z500` or `P500`, point_stat no matched points.
Can you give me some advice?

Yours sincerely

**Comment by JohnHalleyGotway** [comment, marked as answer]
Hello @wangrenz, I see you are getting 0 matched pairs from Point-Stat and are wondering why. There are many possible reasons, including mismatches in variable name, message type, location, time, quality control flags, and vertical levels.

The best place to look for direction is in the log output from Point-Stat. I'd recommend rerunning with at least logging verbosity level 3 (`-v 3` on the command line). You can also write the log messages to a file using the `-log filename` command line option.

Then read through this [FAQ](https://met.readthedocs.io/en/latest/Users_Guide/appendixA.html#q-how-can-i-understand-the-number-of-matched-pairs). You'll find a list of *rejection reason counts* in the log output. The order in which the filtering logic is applied matches the order in which the log messages are written. Reading the counts from the top-to-bottom, you'll find the first non-zero one and that'll tell you where to start looking.

Hope that helps clarify. If you have more questions, please copy/paste those rejection reason counts into your response.

Thanks!

**Reply by JohnHalleyGotway** [reply]
Just wanted to follow up on this discussion to see if questions remain. If not, please feel free to select the "Mark As Answer" button for one of the previous responses, and then I'll lock it to prevent future posts. If I don't hear otherwise in the next week or so, I'll just go ahead and mark it as being answered.

Thanks!
